src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -133,51 +151,36 @@ func DeduceBrokers(
 		if err != nil {
 			log.Trace(
 				"Couldn't read the config file." +
-					" Assuming 127.0.0.1:9092",
+					" Assuming 127.0.0.1:9092.",
 			)
 			log.Debug(err)
 			return []string{"127.0.0.1:9092"}
 		}
 
-		if len(conf.Redpanda.KafkaApi) == 0 {
-			log.Trace(
-				"The config file contains no kafka listeners." +
-					" Empty redpanda.kafka_api.",
+		if len(conf.Rpk.KafkaApi.Brokers) == 0 {
+			log.Debug(
+				"Empty rpk.kafka_api.brokers. Assuming 127.0.0.1:9092.",
 			)
-			return []string{}
+			return []string{"127.0.0.1:9092"}
 		}
 
 		// Add the seed servers' Kafka addrs.
-		if len(conf.Redpanda.SeedServers) > 0 {
-			for _, b := range conf.Redpanda.SeedServers {
-				addr := fmt.Sprintf(
-					"%s:%d",
-					b.Host.Address,
-					conf.Redpanda.KafkaApi[0].Port,
-				)
-				bs = append(bs, addr)
-			}
+		if len(conf.Rpk.KafkaApi.Brokers) > 0 {
```

Comment:
Both of these conditionals are using `conf.Rpk.KafkaApi.Brokers`?

Reply:
You're right, this one can be removed.